src/api/app.py

```diff
@@ -134,11 +184,18 @@ def update_user_info():
 
 @app.route('/api/session', methods=['POST'])
 def log_in():
-    return session_controller.add_session(request.json)
+    session_res = session_controller.add_session(request.json).json
```

Comment:
Me gusta using flask sessions, later we can look to deprecate the session controller

Reply:
:smile: